packages/react/src/primitives/Divider/__tests__/Divider.test.tsx

```diff
@@ -12,7 +12,7 @@ describe('Divider component', () => {
     const divider = (await screen.findByRole('separator')) as HTMLHRElement;
     expect(divider.nodeName).toBe('HR');
     expect(divider.id).toBe(dividerTestId);
-    expect(divider.dataset['size']).toBeUndefined();
```

Comment:
no difference here, they are functionally equivalent, this went from `.toBeUndefined()` -> `.toBe('default')` -> `.toBe(undefined)` just because I forgot that these originally started as `.toBeUndefined()` when I was backing out the changes I had for adding in a default state.

Reply:
Plz use `.toBeUndefined()` to be consistent